src/include/kernel/abi.h

```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 21
+#define SOF_ABI_MINOR 22
```

Comment:
What is this used for? How does this guarantee backward compatibility?

Reply:
This ABI MINOR version is required due to the changes in common dai structure b/w kernel and firmware. This will let you know  both kernel and Firmware should be aligned for use cases to work.
Also some of the Scripts were rejecting the patch for upstream if we don't update ABI version as the changes are updating the dai structures.